docker/dev/install_thrift.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+wget "http://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=thrift/0.13.0/thrift-0.13.0.tar.gz" \
+-O thrift-0.13.0.tar.gz
+tar -xvf ./thrift-0.13.0.tar.gz
+cd thrift-0.13.0
+# Configure
+./configure --prefix=/thrift_install --silent --without-python    \
+  --enable-libtool-lock --enable-tutorial=no --enable-tests=no    \
+  --with-libevent --with-zlib --without-nodejs --without-lua      \
+  --without-ruby --without-csharp --without-erlang --without-perl \
+  --without-php --without-php_extension --without-dart            \
+  --without-haskell --without-go --without-rs --without-haxe      \
+  --without-dotnetcore --without-d --without-qt4 --without-qt5    \
+  --without-java
+make install -j $(nproc)
+# Copy to /usr
+cp -rn /thrift_install/* /usr
```

Comment:
Conventionally the `/usr` folder is for files installed through the package manager. Locally compiled programs should go to `/usr/local`.

Therefore, simply `/usr/local` should be given as the install path and no copying is needed. If you don't define a `--prefix` for the `./configure`, that will be the default value.

Reply:
We have discussed on the weekly meeting, that this could also be a valid approach. However, I have further doubts, as the install folder could also contain symlinks. If those are not relative, but absolute, copying them would result in invalid links. Therefore installing directly into `/usr/local/` would still be preferred.